vendor/bat-native-ledger/src/bat/ledger/internal/logging.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/logging.h"
+
```

Comment:
nit: extra line

Reply:
This is correct as this is the header belonging to this implementation, so following style guide